holmes/core/llm.py

```diff
@@ -424,7 +428,8 @@ def configure_robusta_ai_model(self) -> None:
 
             for model in robusta_models.models:
                 logging.info(f"Loading Robusta AI model: {model}")
-                self._llms[model] = self._create_robusta_model_entry(model)
+                args = robusta_models.models_args.get(model)
+                self._llms[model] = self._create_robusta_model_entry(model, args)
```

Comment:
Since it is urgent, please open sub task on the ticket to add tests. Let's do them after the release. Thanks.